src/migrations/20240118093611-missing-primary-keys.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+        ALTER TABLE project_stats ADD PRIMARY KEY (project);
+        ALTER TABLE api_token_project ADD COLUMN id SERIAL PRIMARY KEY;
```

Comment:
@gastonfournier in this case I would use secret and project here also, if we using combined keys.